### DISCUSSION TITLE
Frustrated with GitHub Support – Considering Migrating to GitLab

### DISCUSSION THREAD

**Discussion by SAHELISAHAA** [community/community#135371] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

**Subject: Frustrated with GitHub Support – Considering Migrating to GitLab**

I’m beyond frustrated right now. My GitHub account (username: mavrick-1) was suspended 12 days ago after I posted about the 10th anniversary of the Student Developer Pack. I immediately submitted a support ticket (Ticket #: 2913646), but I haven’t received a single response.

What’s even more disappointing is that I have a GitHub Pro membership, yet my account was suspended without any communication or explanation. This suspension has shattered all the hard work and contributions I've put into my projects. I've already lost three job opportunities and am now probably going to lose my current job because all essential company updates are stuck in my suspended GitHub account.

At this point, I’m seriously considering switching to a more reliable platform like GitLab, where I can trust that my work won’t just disappear overnight without any support or communication.

Is there anyone out there who has been in a similar situation? Any advice on how to at least save my projects? Also, is there any way to take legal action against GitHub for the immense damage this has caused me? My faith in GitHub support is practically nonexistent right now.

Thanks to GitHub’s workflow, all my efforts have been reduced to nothing.

**Rishi Mondal**

**Comment by ghost** [comment]
I am having the same issue. It is frankly unbelievable. I have been waiting for around 20 days, which feels insane.

username: bearwithtea
Ticket: 2900766

**Comment by Sahelisaha04** [comment]
Same issue

**Comment by LiteBrite82** [comment]
Hi @SAHELISAHAA 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

**Reply by ghost** [reply]
Thank you @LiteBrite82!

Do you know if there is any way to view the queue?

Thank you so much.

**Reply by LiteBrite82** [reply]
sorry, not that I'm aware of.

**Reply by AnLaVN03** [reply]
> Hi @SAHELISAHAA
> 
> Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

Hello there, I am also having the same problem with my AnLaVN account (ticket 2942666) and it is seriously affecting my work, please help me restore access to my account

**Comment by elliotvhill** [comment]
Same issue with me -- it's been 18 business days without any movement at all. I ended up moving to GitLab so I could have my work visible _somewhere_.

Not sure about your situation but I was at least able to export an archive of my repositories (although transferring them over to GitLab hasn't been super intuitive).

I am also job-seeking and this has cost me at least two opportunities that I explicitly know of. This is an abysmal user experience.

**Comment by MAVRICK-1** [comment, marked as answer]
👍

**Comment by Cdrepoman** [comment]
What is this even useful for? This whole git shat seems so overly useless im nauseas is this soley to make it easy for the highly overpaid geeks and nerds to make them appear busy to their bosses ?
